04week/loop.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+// Create a new file called loops.js in the /04week folder of your workbook.
+// Complete each of the following exercises.
+// for loop
+
+// Use a for loop to console.log each item in the array carsInReverse.
+const carsInReverse = ['Ford', 'Chevy', 'Tesla', 'Nissan', 'Toyota'];
+for (i = 0; i < carsInReverse.length; i++) {
```

Comment:
i is still a variable that hasn't been declared before, so you need to use `let`